mxcube3/routes/qutils.py

```diff
@@ -226,6 +226,11 @@ def _handle_dc(sample_id, node):
     parameters.pop('centred_position')
     queueID = node._node_id
     enabled, state = get_node_state(queueID)
+    parameters['subdir'] = parameters['path'].split(mxcube.session.get_base_image_directory())[1][1:]
+    try:
+        limsres = mxcube.rest_lims.get_dc(node.id)
+    except:
```

Comment:
Would be really great to display the traceback or to raise an error if a real error happens, at the moment every error is eaten up by `except:`. Maybe it is enough to test for `if mxcube.rest_lims:` ?